src/bn256/common.rs

```diff
@@ -99,18 +99,30 @@ macro_rules! common_field {
 
                 $field([d0, d1, d2, d3])
             }
+
+            /// Attempts to convert a little-endian byte representation of
+            /// a scalar into a `Fr`, failing if the input is not canonical.
+            pub fn from_bytes(bytes: &[u8; 32]) -> CtOption<$field> {
+                <Self as ff::PrimeField>::from_repr(*bytes)
+            }
+
+            /// Converts an element of `Fr` into a byte representation in
+            /// little-endian byte order.
+            pub fn to_bytes(&self) -> [u8; 32] {
+                <Self as ff::PrimeField>::to_repr(self)
+            }
```

Comment:
Seems we do have wrappers again! 
But it's sad to see that we get a `CtOption` for the `from_bytes`...  I understand why, but it makes using it much less attractive.